src/main/java/uk/gov/hmcts/reform/et/service/ServiceHearingsService.java

```diff
@@ -28,6 +28,9 @@ public class ServiceHearingsService {
     @Value("${et.hmctsServiceId}")
     private String hmctsServiceId;
 
+    @Value("${case-details-url.exui}")
```

Comment:
Minior but shouldnt it be the other way around ? exui.case-details-url

Reply:
Perhaps, but we use this same variable in et-ccd-callbacks and et-sya-api so I wanted to keep it consistent